doc/source/user/absolute_beginners.rst

```diff
@@ -1685,6 +1687,7 @@ With Matplotlib, you have access to an enormous number of visualization options.
   >>> Z = np.sin(R)
 
   >>> ax.plot_surface(X, Y, Z, rstride=1, cstride=1, cmap='viridis')
+  <mpl_toolkits.mplot3d.art3d.Poly3DCollection object at 0x...>
```

Comment:
This is what I was talking about. This is 4 lines of random output that just detracts from the example. We seem to have a couple more, but not many luckily:
```
$ rg "matplotlib.lines"
doc/source/reference/routines.polynomials.classes.rst
338:    [<matplotlib.lines.Line2D object at 0x2136c10>]
341:    [<matplotlib.lines.Line2D object at 0x1cf2890>]

numpy/core/function_base.py
112:    [<matplotlib.lines.Line2D object at 0x...>]
114:    [<matplotlib.lines.Line2D object at 0x...>]
267:    [<matplotlib.lines.Line2D object at 0x...>]
269:    [<matplotlib.lines.Line2D object at 0x...>]
380:    [<matplotlib.lines.Line2D object at 0x...>]
382:    [<matplotlib.lines.Line2D object at 0x...>]

numpy/lib/function_base.py
1514:    [<matplotlib.lines.Line2D object at 0x...>]
1516:    [<matplotlib.lines.Line2D object at 0x...>]
2904:    [<matplotlib.lines.Line2D object at 0x...>]
2923:    [<matplotlib.lines.Line2D object at 0x...>]
3013:    [<matplotlib.lines.Line2D object at 0x...>]
3032:    [<matplotlib.lines.Line2D object at 0x...>]
3116:    [<matplotlib.lines.Line2D object at 0x...>]
3135:    [<matplotlib.lines.Line2D object at 0x...>]
3218:    [<matplotlib.lines.Line2D object at 0x...>]
3235:    [<matplotlib.lines.Line2D object at 0x...>]
3497:    [<matplotlib.lines.Line2D object at 0x...>]
3514:    [<matplotlib.lines.Line2D object at 0x...>]
3603:    [<matplotlib.lines.Line2D object at 0x...>]

numpy/fft/_pocketfft.py
207:    [<matplotlib.lines.Line2D object at 0x...>, <matplotlib.lines.Line2D object at 0x...>]
304:    [<matplotlib.lines.Line2D object at ...>]
306:    [<matplotlib.lines.Line2D object at ...>]
```
I'd much prefer not to see these lines. It's best just not to see them (the example runner should ignore them), second best to add `_ = ` to the plot calls to swallow the output.

Reply:
For NumPy we don't have many plots in the docs, so it's not a big deal. In SciPy this would be a lot of noise.

Reply:
This output line can still be removed.